scaloid-common/src/main/st/org/scaloid/common/app.scala

```diff
@@ -96,7 +108,17 @@ trait SActivity extends Activity with SContext with TraitActivity[SActivity] wit
 
   protected override def onCreate(b: Bundle) {
     super.onCreate(b)
-    onCreateBodies.foreach(_ ())
+    AppHelpers.createBundle.withValue(Option(b)) {
+      onCreateBodies.foreach(_ ())
+    }
+  }
+
+  def onCreate(body: Option[Bundle] => Any) = {
+    val el = { () =>
+      body(AppHelpers.createBundle.value)
```

Comment:
My bad, I just saw the addition without noticing you didn't remove ` onCreate(body: => Any)` that was in the `Creatable` trait.